sensor_msgs/srv/SnapshotImage.srv

```diff
@@ -0,0 +1,17 @@
+# Image as a service
+
+# REQUEST
+
+# exposure in milliseconds (0 indicates autoexposure) 
+uint32 exposure
```

Comment:
There's the same issue here. Unless you're going to provide all the settings providing just one doesn't make much sense.

Reply:
Yes, I agree. Providing one parameter is quite arbitrary and not general. I can modify both srv with an std_msgs/Empty in the request side. 
Another possibility could be to request with an std_msgs/UInt8 indicating the number of snapshots to take, and reply with a vector of either sensor_msgs/PointCloud2 or sensor_msgs/Image.
By the way, thanks for reviewing this PR!